core/app/models/spree/reimbursement_type/reimbursement_helpers.rb

```diff
@@ -44,7 +44,25 @@ def create_credit(reimbursement, unpaid_amount, simulate)
     end
 
     def create_creditable(reimbursement, unpaid_amount)
-      Spree::Reimbursement::Credit.default_creditable_class.new(reimbursement: reimbursement, amount: unpaid_amount)
+      category = Spree::StoreCreditCategory.default_reimbursement_category(category_options(reimbursement))
+      Spree::StoreCredit.new(store_credit_params(category, reimbursement, unpaid_amount))
     end
+
+    def store_credit_params(category, reimbursement, unpaid_amount)
+      {
+        user: reimbursement.order.user,
+        amount: unpaid_amount,
+        category: category,
+        created_by: Spree::StoreCredit.default_created_by,
+        memo: "Refund for uncreditable payments on order #{reimbursement.order.number}",
+        currency: reimbursement.order.currency
+      }
+    end
+
+    # overwrite if you need options for the default reimbursement category
+    def category_options(reimbursement)
```

Comment:
Unused method argument - `reimbursement`. If it's necessary, use `_` or `_reimbursement` as an argument name to indicate that it won't be used. You can also write as `category_options(*)` if you want the method to accept any arguments but don't care about them.